webproject/src/main/java/com/google/sps/services/implementations/UserRoomStatusServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.services.implementations;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import com.google.sps.authentication.AuthenticationHandler;
+import com.google.sps.dataManagers.UserRoomManager;
+import com.google.sps.proto.UserRoomStatusProto.UserRoomStatusRequest;
+import com.google.sps.services.interfaces.UserRoomStatusService;
+import com.google.sps.util.FirebaseUtil;
+
+@Singleton
+public class UserRoomStatusServiceImpl implements UserRoomStatusService {
+    private AuthenticationHandler authenticationHandler;
+    private UserRoomManager userRoomManager;
+
+    @Inject
+    public UserRoomStatusServiceImpl(AuthenticationHandler authenticationHandler, UserRoomManager userRoomManager) {
+        this.userRoomManager = userRoomManager;
+        this.authenticationHandler = authenticationHandler;
+    }
+
+    @Override
+    public boolean execute(UserRoomStatusRequest getJoinRequest) throws IOException {
```

Comment:
lets have a UserRoomStatusResponse here as well. we can put is_joined attr inside the proto.
In the future we might want to have more data inside the status like whether the room is paid, etc.

Reply:
done